server/sonar-server/src/main/java/org/sonar/server/usergroups/ws/UpdateAction.java

```diff
@@ -65,52 +71,58 @@ public void define(NewController context) {
       .setResponseExample(getClass().getResource("example-update.json"))
       .setSince("5.2");
 
-    action.createParam(PARAM_ID)
+    action.createParam(PARAM_GROUP_ID)
       .setDescription("Identifier of the group.")
       .setExampleValue("42")
       .setRequired(true);
 
-    action.createParam(PARAM_NAME)
-      .setDescription(String.format("New name for the group. A group name cannot be larger than %d characters and must be unique. " +
-        "The value 'anyone' (whatever the case) is reserved and cannot be used.", GROUP_NAME_MAX_LENGTH))
+    action.createParam(PARAM_GROUP_NAME)
+      .setDescription(String.format("New optional name for the group. A group name cannot be larger than %d characters and must be unique. " +
+        "The value 'anyone' (whatever the case) is reserved and cannot be used. If value is empty or not defined, then name is not changed.", GROUP_NAME_MAX_LENGTH))
       .setExampleValue("sonar-users");
 
-    action.createParam(PARAM_DESCRIPTION)
-      .setDescription(String.format("New description for the group. A group description cannot be larger than %d characters.", DESCRIPTION_MAX_LENGTH))
+    action.createParam(PARAM_GROUP_DESCRIPTION)
+      .setDescription(String.format("New optional description for the group. A group description cannot be larger than %d characters. " +
+        "If value is not defined, then description is not changed.", DESCRIPTION_MAX_LENGTH))
       .setExampleValue("Default group for new users");
   }
 
   @Override
   public void handle(Request request, Response response) throws Exception {
     userSession.checkLoggedIn().checkPermission(GlobalPermissions.SYSTEM_ADMIN);
 
-    Long groupId = request.mandatoryParamAsLong(PARAM_ID);
-    String name = request.param(PARAM_NAME);
-    String description = request.param(PARAM_DESCRIPTION);
-
     DbSession dbSession = dbClient.openSession(false);
     try {
+      Long groupId = request.mandatoryParamAsLong(PARAM_GROUP_ID);
       GroupDto group = dbClient.groupDao().selectById(dbSession, groupId);
-      if (group == null) {
-        throw new NotFoundException(String.format("Could not find a user group with id '%s'.", groupId));
-      }
-      String oldName = group.getName();
-      if (name != null) {
-        groupUpdater.checkNameIsUnique(name, dbSession);
-        groupUpdater.validateName(name);
-        group.setName(name);
-        updateDefaultGroupIfNeeded(dbSession, oldName, name);
+      checkFound(group, "Could not find a user group with id '%s'.", groupId);
+      Optional<OrganizationDto> org = dbClient.organizationDao().selectByUuid(dbSession, group.getOrganizationUuid());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) NullPointerException might be thrown as 'group' is nullable here [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2259)